dask/dataframe/io/parquet.py

```diff
@@ -390,7 +390,7 @@ def to_parquet(path, df, compression=None, write_index=None, has_nulls=True,
 
     if append:
         pf = fastparquet.api.ParquetFile(path, open_with=myopen, sep=sep)
-        if pf.file_scheme != 'hive':
+        if pf.file_scheme not in ['hive', 'empty', 'flat']:
```

Comment:
Is there a `"flat"` scheme somewhere? Am only finding `"empty"` and `"simple"`. Is it suppose to be `"simple"`?

Reply:
"flat" is new, when there are multiple files, but all are in one directory